CHANGES/6718.removal

```diff
@@ -0,0 +1 @@
+Stop using artifacts when importing collections
```

Comment:
I think this could be `6718.misc` instead because it's not a change a user could perceive as different.